scripts/actions/release-notes.py

```diff
@@ -32,7 +35,11 @@
   except AttributeError:
     pass
 
+# Limit the characters in the notes to 25000 as per api restrictions
+result = result[0:24999]
+
+# Encode the string to escape characters
+result = result.replace('"','\\"')
```

Comment:
i think you may wanna swap these lines. if we replace one character with two, that could send us over the limit again.

Reply:
Ah yes i was trying to be clever and not have to replace characters if they were over a certain limit already 🥇